File: text/php/delete/text_delete.php
#! /usr/bin/php
<?php
// --------------------------------------------------------------------
/*

	php/delete/text_delete.php

					Sep/15/2016

*/
// --------------------------------------------------------------------
$path="/var/www/data_base/common/php_common";
set_include_path (get_include_path() . PATH_SEPARATOR . $path);


include "text_manipulate.php";
include "file_io.php";
// --------------------------------------------------------------------
$file_in = $argv[1];
$id_in = $argv[2];

fputs (STDERR,"*** 開始 ***\n");

print	$file_in . "\n";
print	$id_in . "\n";

$dict_aa = text_read_proc ($file_in);
$dict_bb = dict_delete_proc ($dict_aa,$id_in);

text_write_proc ($file_in,$dict_bb);

fputs (STDERR,"*** 終了 ***\n");
/* -------------------------------------------------------------------- */
?>

File: text/lua/delete/text_delete.lua
#! /usr/bin/lua
-- ----------------------------------------------------
--	text_delete.lua
--
--				Jun/22/2011
--
-- ----------------------------------------------------
require	('text_manipulate')
-- ----------------------------------------------------
print ("*** 開始 ***")

local file_txt=arg[1]
local id_in= arg[2]

print (id_in)

dict_aa = text_read_proc (file_txt)

dict_bb = dict_delete_proc (dict_aa,id_in)

dict_display_proc (dict_bb)

text_write_proc (file_txt,dict_bb)

print ("*** 終了 ***")
-- ----------------------------------------------------
